frontend-react/src/components/Content/Templates/IASideNavTemplate.tsx

```diff
@@ -0,0 +1,40 @@
+import { Navigate, Route, Routes } from "react-router-dom";
+
+import { ContentDirectory, getDirectoryElement } from "../MarkdownDirectory";
+
+import GeneratedSideNav from "./GeneratedSideNav";
+
+export interface IASideNavProps {
+    directories: ContentDirectory[];
+}
+
+const IASideNavTemplate = ({ directories }: IASideNavProps) => {
+    return (
+        <section className="grid-container tablet:margin-top-6 margin-bottom-5">
+            <div className="grid-row grid-gap">
+                <section className="tablet:grid-col-4 margin-bottom-6">
+                    <GeneratedSideNav directories={directories} />
+                </section>
+                <section className="tablet:grid-col-8 usa-prose rs-documentation">
+                    <Routes>
+                        <Route
+                            path={"/"}
+                            element={
+                                <Navigate to={"/product/overview"} replace />
```

Comment:
Side navigation templates include their own router because of how side navigation component logic works.

Reply:
yeah, I'm mainly interested in getting rid of the hardcoded redirect here, and it seems like replacing this with an IARouter would provide us with the ability to make that dynamic. If you'd like to tackle that later, can we make sure that we card it so that it doesn't surprise us when we want to use this template for something other than a product page?

Reply:
This is a pre-index IA template, and the only one we needed for the MVP. When we did updates a few sprints ago, we had an added requirement of index pages. This template doesn't fit that model, so the IARouter that was built for that specific need doesn't fit into the way this template renders. An index is its own entire element, so for this template to adapt the new IARouter, rather than state switching to show new content within a single element, we'd have to have a unique element to render for _each section_, included with its own uniquely rendered sidenav. If we want to refactor it outside of the scope of adopting react-router we can ticket it.